### PR TITLE
More fixes

### LINKED ISSUE #356: Switch to bicubic sampling on zoom levels less than 100%
nearest neighbor looks ugly when you zoom out

### LINKED ISSUE #360: Crash when resizing the canvas leads to a layer going completely out of frame
1. New canvas
2. Draw a dot in the bottom right corner
3. Resize canvas to half the size

### PR DESCRIPTION
**Pull request by Equbuxu**
- [x] Fix #360 
- [x] Fix various clip canvas bugs
- [x] Startup window is no longer closed when you click outside of it
- [x] Fix #356
- [x] Change the zoom step size to allow zooming to exact multiples of 100%
- [x] Add bicubic sampling to file previews, layer previews, and navigator

### CODE REVIEW

File: PixiEditor/Helpers/Converters/ZoomLevelToBitmapScalingModeConverter.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using System.Windows.Media;
+
+namespace PixiEditor.Helpers.Converters
+{
+    internal class ZoomLevelToBitmapScalingModeConverter : SingleInstanceConverter<ZoomLevelToBitmapScalingModeConverter>
+    {
+        public override object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            double zoomLevel = (double)value;
+            Debug.WriteLine(zoomLevel);
```
Comment (flabbet) [inline PixiEditor/Helpers/Converters/ZoomLevelToBitmapScalingModeConverter.cs:13]: reee

**Review by flabbet — CHANGES_REQUESTED** [review]

File: PixiEditor/Helpers/Converters/ZoomLevelToBitmapScalingModeConverter.cs (hunk 1 above)
Comment (Equbuxu) [reply PixiEditor/Helpers/Converters/ZoomLevelToBitmapScalingModeConverter.cs:13]: yep, I know :D

**Comment by flabbet** [comment]
https://user-images.githubusercontent.com/25402427/15483773
[clip_broken.zip](https://github.com/PixiEditor/PixiEditor/files/8103763/clip_broken.zip)
4-9efc1c47-95f0-4640-81bc-f4d052d339c4.mp4

**Comment by flabbet** [comment]
Seems to be working pretty nice now

File: PixiEditor/Views/Dialogs/HelloTherePopup.xaml.cs [hunk 2]
```diff
@@ -28,6 +28,7 @@ public partial class HelloTherePopup : Window
 
         public bool RecentlyOpenedEmpty { get => (bool)GetValue(RecentlyOpenedEmptyProperty); set => SetValue(RecentlyOpenedEmptyProperty, value); }
 
+
```
Comment (flabbet) [inline PixiEditor/Views/Dialogs/HelloTherePopup.xaml.cs:31]: any reason of this?

**Review by flabbet — APPROVED** [review]

**Pull request merged**